host-configs/lc-builds/toss4/corona_sycl.cmake

```diff
@@ -0,0 +1,25 @@
+###############################################################################
+# Copyright (c) 2016-22, Lawrence Livermore National Security, LLC
+# and RAJA project contributors. See the RAJA/LICENSE file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+###############################################################################
+
+set(RAJA_COMPILER "RAJA_COMPILER_CLANG" CACHE STRING "")
+
+set(CMAKE_CXX_COMPILER "clang++" CACHE PATH "")
```

Comment:
@artv3 @rhornung67 Should I hard-code this path to the installed compiler on `corona` (i.e. `/usr/workspace/raja-dev/clang_sycl_hip_gcc10.2.1_rocm5.1.0/bin/clang++`)? This would eliminate the need for the user to alter their `$PATH`.